samples/Cordova/Backend/Backend.xproj

```diff
@@ -7,12 +7,12 @@
   <Import Project="$(VSToolsPath)\DotNet\Microsoft.DotNet.Props" Condition="'$(VSToolsPath)' != ''" />
   <PropertyGroup Label="Globals">
     <ProjectGuid>1f7aaac4-26e8-408a-aaf2-37b9ac35ad5c</ProjectGuid>
-    <RootNamespace>Backend</RootNamespace>
+    <RootNamespace>Mvc.Server</RootNamespace>
```

Comment:
Curious: any particular reason to rename this project? If you really want to change it, consider also updating the folder name.